gnucash/import-export/qif-imp/CMakeLists.txt

```diff
@@ -8,6 +8,18 @@ set (qif_import_SOURCES
     gnc-plugin-qif-import.c
 )
 
+set (qif_import_remote_HEADERS
```

Comment:
No need for a separate set of cmake variables, just add the files to `qif_import_FOO`.